crates/oci/src/client.rs

```diff
@@ -1141,15 +1141,13 @@ mod test {
                 .expect("should have version annotation")
         );
         assert!(
-            annotations
-                .get(oci_distribution::annotations::ORG_OPENCONTAINERS_IMAGE_DESCRIPTION)
-                .is_none(),
+            !annotations
```

Comment:
This and the next diff seem unrelated to all things Azure.  Maybe your clippy is more recent than the one in CI?  (I also noticed some format changes on the OTel attributes which suggested a toolchain having new opinions about existing code.)

Reply:
Happy to revert the attribute changes. Let me know. Pretty sure it was just RustRover doing its auto format thing.

As for the annotations, IIRC that was a clippy error with the Rust version I am using.

Reply:
You and your fancy "up-to-date" Rust versions.  Yeah I'm not too worried.  Thanks - I'll merge once Radu has had a chance to look.